src/config/constants/tokens.ts

```diff
@@ -2000,7 +2000,7 @@ export const mainnetTokens = defineTokens({
     'BitTorrent',
     'https://bittorrent.com/',
   ),
-  era: new Token(MAINNET, '0x6f9F0c4ad9Af7EbD61Ac5A1D4e0F2227F7B0E5f9', 18, 'ERA', 'Era Token', 'https://www.era7.io/'),
+  era: new Token(MAINNET, '0x6f9F0c4ad9Af7EbD61Ac5A1D4e0F2227F7B0E5f9', 18, 'ERA', 'Era7', 'https://www.era7.io/'),
```

Comment:
The token name is officially Era Token, let's not change it here I think.
https://bscscan.com/address/0x6f9F0c4ad9Af7EbD61Ac5A1D4e0F2227F7B0E5f9#readContract